api-gateway/src/test/java/com/petclinic/bffapigateway/domainclientlayer/VisitsServiceClientIntegrationTest.java

```diff
@@ -39,6 +48,10 @@ class VisitsServiceClientIntegrationTest {
     private VisitsServiceClient visitsServiceClient;
     private static MockWebServer server;
     private static ObjectMapper objectMapper = new ObjectMapper();
+    @Autowired
```

Comment:
Why are webTestClient and webClientBuilder being added?

Reply:
removed it!